sigs.yaml

```diff
@@ -28,5 +28,30 @@ workinggroups:
   contact:
     slack: kubernetes-operators
     mailing_list: https://groups.google.com/forum/#!forum/operator-framework-wg-ci
+- dir: wg-olm
+  name: OLM
+  mission_statement: >
+    To organize community input for the Operator Lifecycle Manager project and it's
```

Comment:
nit `its` is the possessive, `it's` is a conjunction. Yay English! :)

Reply:
Resolved in the latest version of the PR.